src/common/constants/constants.js

```diff
@@ -0,0 +1 @@
+export const SYNTHETIC_USER_ID = '06c3485f-631c-427e-bade-5e763636c470';
```

Comment:
it's located in common constants so can not be very clear to understand to which functionality this constant belongs.
Propose to rename to CLAIMING_JOB_SYNTETIC_USER_ID